modules/videoio/src/cap_v4l.cpp

```diff
@@ -261,7 +261,12 @@ struct buffer
     size_t  length;
     // This is dequeued buffer. It used for to put it back in the queue.
     // The buffer is valid only if capture->bufferIndex >= 0
-    v4l2_buffer buffer;
+    v4l2_buffer v4l2_buf;
+
+    buffer() : start(NULL), length(0)
+    {
+        memset(&v4l2_buf, 0, sizeof(v4l2_buf));
```

Comment:
`v4l2_buf = v4l2_buffer()` this initialization style is used in this cpp file. Please, don't mix it.

Reply:
If you care about variable names, I could bring everything in this file to a single style, including renaming ambiguous entities

Reply:
Renamed type instead.
Using `v4l2_buffer()` initializer.